.github/workflows/feature_request.yml

```diff
@@ -0,0 +1,24 @@
+name: Feature request label added.
+
+on:
+  issues:
+    types: [labeled]
+
+permissions:
+  issues: write
+
+jobs:
+  project:
+    name: Feature request message
+    runs-on: ubuntu-latest
+    if: ${{ github.event.label.name == 'feature-request' }}
+
+    steps:
+      - uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
```

Comment:
```suggestion
```
optimization: you should be able to specify the repo in the gh cli command and not need to perform the checkout (in conjunction with other suggestion below)

Reply:
Nice! TIL